plugins/woocommerce-blocks/assets/js/blocks/product-query/variations/related-products.tsx

```diff
@@ -117,31 +117,30 @@ export const INNER_BLOCKS_TEMPLATE: InnerBlockTemplate[] = [
 	],
 ];
 
-registerProductBlockType( {
-	blockName: QUERY_LOOP_ID,
-	blockMetadata: {},
-	blockSettings: {
-		description: __( 'Display related products.', 'woocommerce' ),
-		name: 'Related Products Controls',
```

Comment:
I don't even know why this is here, as far as I know `name` is the block nomenclature not the title. But what makes this move confusing is that `title` is also defined right below it.